utilities/test_suite/HIP/Tensor_hip.cpp

```diff
@@ -126,7 +123,7 @@ int main(int argc, char **argv)
         printf("\nReduction Kernels don't have outputFormatToggle! Please input outputFormatToggle = 0\n");
         return -1;
     }
-    else if(batchSize > MAX_BATCH_SIZE)
+    if(batchSize > MAX_BATCH_SIZE)
```

Comment:
why this change is made?
Please check the test suite changes once

Reply:
This was done by mistake, reverted it